src/Nancy.Bootstrappers.Ninject/NinjectNancyBootstrapper.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Nancy.Bootstrappers.Ninject
+﻿using Nancy.Configuration;

```

Comment:
I blame weird conventions. A DotSettings is the only way this'll happen automatically :wink: